cmake/public/cuda.cmake

```diff
@@ -232,7 +232,7 @@ if(CAFFE2_USE_CUSPARSELT)
     target_link_libraries(torch::cusparselt INTERFACE ${CUSPARSELT_LIBRARY_PATH})
   endif()
 else()
-  message(STATUS "USE_CUSPARSELT is set to 0. Compiling without cuSPARSELt support")
+  message(STATUS "CAFFE2_USE_CUSPARSELT is set to 0. Compiling without cuSPARSELt support")
 endif()
 
 if(USE_CUDSS)
```

Comment:
That one did not have a `CAFFE2_` prefix which is unfortunate, but changing it would break backward compatibility so I didn't update here.